### PR TITLE
Feature/transaction screen

### PR DESCRIPTION
**Pull request by Mikitka101**
Implement UI for TransactionDetailsScreen and TransactionListScreen

### CODE REVIEW

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import com.nikitayasiulevich.bankingapp.presentation.ui.theme.DarkGrey
+import com.nikitayasiulevich.bankingapp.presentation.ui.theme.LightGrey
+import kotlin.random.Random
+
+@Composable
+fun MainScreen(
+    //navController: NavController,
+    //transactionsViewModel: TransactionListViewModel,
+    //accountsViewModel: TransactionListViewModel
+) {
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt:50]: remove unused code

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    val transactionsFromViewModel = mutableListOf<Transaction>()
+    for (i in 0 until 5) {
+        val item = Transaction(
+            id = i.toString(),
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt:54]: for cannot be on UI

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                Constants.TRANSACTION_STATUS_IN_PROGRESS,
+                Constants.TRANSACTION_STATUS_DECLINED
+            )[Random.nextInt(0, 3)],
+            amount = Random.nextInt(0, 9999) + Random.nextFloat()
+        )
+        transactionsFromViewModel.add(item)
+    }
+    val accountsFromViewModel = mutableListOf<Account>()
+    for (i in 0 until 5) {
+        val item = Account(
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt:68]: the same

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.nikitayasiulevich.bankingapp.domain.model.Transaction
+
+@Composable
+fun TransactionDetailsScreen(
+    transaction: Transaction?,
+    //navController: NavController,
+    //transactionsViewModel: TransactionListViewModel
+) {
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt:35]: unused code

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    var id by remember { mutableStateOf("") }
+    var company by remember { mutableStateOf("") }
+    var status by remember { mutableStateOf("") }
+    var date by remember { mutableStateOf("") }
+    var amount by remember { mutableStateOf("") }
+    var isReadOnly by remember { mutableStateOf(false) }
+
+    var funOnButtonClick: () -> Unit = {
+        //TODO (save new transaction)
+    }
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt:45]: TODO?

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+    transaction?.let { tr ->
+        id = tr.id
+        company = tr.company
+        date = tr.date
+        status = tr.status
+        amount = "$${tr.amount}"
+        isReadOnly = true
+        funOnButtonClick = {
+            //TODO (navigate to main screen)
+        }
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt:56]: the same

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionListScreen.kt [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+import com.nikitayasiulevich.bankingapp.presentation.components.TransactionListItem
+import com.nikitayasiulevich.bankingapp.presentation.ui.theme.DarkGrey
+import com.nikitayasiulevich.bankingapp.presentation.ui.theme.LightGrey
+import kotlin.random.Random
+
+@Composable
+fun TransactionListScreen(
+    //navController: NavController,
+    //transactionsViewModel: TransactionListViewModel
+) {
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionListScreen.kt:45]: unused

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionListScreen.kt [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+    val transactionsFromViewModel = mutableListOf<Transaction>()
+    for (i in 0 until 12) {
+        val item = Transaction(
+            id = i.toString(),
```
Comment (Razor0744) [inline app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionListScreen.kt:49]: for

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt (hunk 1 above)
Comment (Mikitka101) [reply app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt]: I did

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt (hunk 2 above)
Comment (Mikitka101) [reply app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt]: I did

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt (hunk 3 above)
Comment (Mikitka101) [reply app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/MainScreen.kt]: I did

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt (hunk 4 above)
Comment (Mikitka101) [reply app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt]: I did

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt (hunk 5 above)
Comment (Mikitka101) [reply app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt]: I did

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt (hunk 6 above)
Comment (Mikitka101) [reply app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionDetailsScreen.kt]: I did

File: app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionListScreen.kt (hunk 7 above)
Comment (Mikitka101) [reply app/src/main/java/com/nikitayasiulevich/bankingapp/presentation/TransactionListScreen.kt]: I did

**Pull request merged**